src/client/assets/templates/cla.html

```diff
@@ -19,14 +19,14 @@ <h4>{{titleMsg()}}</h4>
             <div class="cla-box well">
                 <div ng-bind-html="cla"></div>
             </div>
-            <label ng-class="required" ng-show="linkedItem.privacyPolicy">
+            <label ng-class="required" ng-show="linkedItem.privacyPolicy && (!hasCustomFields || user.value)">
                 <input ng-disabled="signed" ng-model="privacyPolicyAccepted" type="checkbox" /> I accept that the information I provide to sign this CLA will be maintained in accordance with project
                 owner's
                 <a class="highlighted-link" ng-href="{{linkedItem.privacyPolicy}}" target="space">privacy policy</a> *
             </label>
             <form class="row" ng-if="hasCustomFields">
                 <p>
-                    <button class="btn btn-info btn-lg" ng-show="hasCustomFields && !signed && linkedItem && claText && !user.value" ng-click="signIn()">Sign in with GitHub to agree</button>
+                    <button class="btn btn-info btn-lg" ng-show="hasCustomFields && !signed && linkedItem && claText && !user.value" ng-click="signIn()">Login with GitHub to continue</button>
```

Comment:
This is the major change.  Clicking this button does _not_ agree, it just signs you in so you can see the custom fields, and enables the real "I agree" button.

Reply:
I agree that this change (changing "to agree" to "to continue") fixes the part that I found confusing.